src/app/api/collections/utils.py

```diff
@@ -0,0 +1,19 @@
+from typing import Generic, TypeVar
+
+from elasticsearch_dsl.response import Response
+from glom import glom
+
+from app.api.collections.models import CollectionNode, MissingMaterials
+
+T = TypeVar("T", CollectionNode, MissingMaterials)
+
+
+def map_elastic_response_to_model(
+    response: Response, specs: dict, model: Generic[T]
+) -> list[T]:
+    collections = []
+    for hit in response:
+        parsed_entry = glom(hit.to_dict(), specs)
+        if parsed_entry["title"] is not None:
```

Comment:
why would there be entries with title = None? Would that be a bug in the elastic query? Eventually add a comment why those scenarios do not matter.

Reply:
Ayayay, outdated logic, I cleaned it :)